fms-extras-requirements.txt

```diff
@@ -0,0 +1,9 @@
+# This requirement files can be used for development purposes, as it points to an
+# unreleased version (the main branch) for the ibm-fms package
+# It pins all other fms-extra dependencies to a specific version
+# for repeatable installs
+
+torch==2.2.0
+transformers==4.35.0
```

Comment:
ibm-fms depends on these too right? I wonder if we should just get these indirectly as dependencies of ibm-fms?

Reply:
That's a good point. 

We do have those dependencies in https://github.com/foundation-model-stack/fms-extras/blob/fdb1636de4261fd4102da659ab45d3fcc33fe8ef/setup.py#L13C1-L16C32 though (plus `accelerate` which I had missed).

This package might have its specific requirements on those libraries eventually, possibly stricter than FMS, but we could add them when needed instead of now.

Reply:
I updated this to only include `accelerate` as pinned dependency